hack/bootstrap-cluster.sh

```diff
@@ -45,21 +43,26 @@ kubectl patch argocd/openshift-gitops -n openshift-gitops -p '{"spec":{"rbac":{"
 
 echo 
 echo "Add Role/RoleBindings for OpenShift GitOps:"
-kustomize build $ROOT/openshift-gitops/cluster-rbac | kubectl apply -f -
+kubectl apply --kustomize $ROOT/openshift-gitops/cluster-rbac
 
 echo
-if [ "$MODE" == "" ] || [ "$MODE" == "upstream" ]; then
-    echo "Setting Cluster Mode: Upstream"
-    kubectl apply -f $ROOT/argo-cd-apps/app-of-apps/all-applications-staging.yaml
-elif [ "$MODE" == "development" ]; then
-    echo "Setting Cluster Mode: Development"
-    $ROOT/hack/development-mode.sh
-elif [ "$MODE" == "preview" ]; then
-    echo "Setting Cluster Mode: Preview"
-    $ROOT/hack/preview.sh
-fi
+echo "Setting Cluster Mode: ${MODE:-Upstream}"
+case $MODE in
+    ""|"upstream")
+        kubectl apply -f $ROOT/argo-cd-apps/app-of-apps/all-applications-staging.yaml ;;
+    "development")
+        $ROOT/hack/development-mode.sh ;;
+    "preview")
+        $ROOT/hack/preview.sh ;;
+esac
 
-ARGO_CD_URL="https://$(kubectl get route/openshift-gitops-server -n openshift-gitops -o template --template={{.spec.host}})"
+printf -v ARGO_CD_URL "https://%s" \
+    $(
+        kubectl get route/openshift-gitops-server \
+        -n openshift-gitops \
+	-o template \
+	--template={{.spec.host}}
+    )
```

Comment:
Much harder for me to read, but it's ok.

Reply:
This could be refactored, or even dropped.
If it burns your eyes I'd rather just drop it?

Reply:
Compromise?
```
ARGO_CD_ROUTE=$(kubectl get \
                 -n openshift-gitops \
                 -o template \
                 --template={{.spec.host}} \
                 route/openshift-gitops-server \
               )
ARGO_CD_URL="https://$ARGO_CD_ROUTE"
```

Reply:
I like this. Updated the PR to include the ARGO_CD_ROUTE as shown.